modules/core/src/v2/coins/trx.ts

```diff
@@ -325,13 +325,8 @@ export class Trx extends BaseCoin {
   getExtraPrebuildParams(buildParams: any, callback?: NodeCallback<any>): Bluebird<any> {
     return co<any>(function* () {
       if (buildParams.recipients[0].data && buildParams.feeLimit) {
-        const recipients: any[] = [{ ...buildParams.recipients[0] }];
-        recipients[0].feeLimit = buildParams.feeLimit;
-        return {
-          recipients,
-        };
+        buildParams.recipients[0].feeLimit = buildParams.feeLimit;
       }
-      return {};
```

Comment:
Why is the return type changing? I think we need this to return the new build params: https://github.com/BitGo/BitGoJS/blob/cc358bde41e46ade21e744ad710ef4610652cb78/modules/core/src/v2/wallet.ts#L1687

Reply:
i ve removed those returns and change the buildParams itself because the data was lost on the way to the send().

Reply:
Does this actually work? Have you tested the complete prebuild flow?

Reply:
Yes, we have tested the complete flow of contract calls on wallet-platform and trust admin tool with @pviolasimtlix